action.yml

```diff
@@ -33,11 +33,13 @@ inputs:
 outputs:
   cache-hit:
     description: 'A boolean value to indicate an exact match was found for the primary key'
+  save-always-d18d746b9:
```

Comment:
Give the new output a unique name to reduce conflicts when checked in `post-if`.